discovery/cmd/openstack-discoverer/main.go

```diff
@@ -218,10 +220,29 @@ func main() {
 	log.Info("Starting reconciler")
 	go reconciler.Run(stopCh)
 
+	go func() {
+		http.HandleFunc("/healthz", healthzHandler)
+		log.Fatal(http.ListenAndServe("127.0.0.1:8080", nil))
+		<-stopCh
+		log.Info("Shutting down healthz endpoint...")
+	}()
+
 	<-stopCh
 	log.Info("Stopped OpenStack discoverer")
 }
 
+func healthzHandler(w http.ResponseWriter, r *http.Request) {
+	_, err := reconciler.ProjectLister.ListProjects()
```

Comment:
Is it customary to discard this error? I am wondering if it makes sense to either log it or print it out in the HTTP response

Reply:
Yeah, that makes sense to me. I think we can move forward with discarding this error.